src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -1230,4 +1230,36 @@ theorem shiftRight_eq_ushiftRightRec (x : BitVec w₁) (y : BitVec w₂) :
   · simp [of_length_zero]
   · simp [ushiftRightRec_eq]
 
+/- ### umod -/
+
+theorem getElem_umod_of_lt {n d : BitVec w} (hi : i < w) (hd : 0#w < d) :
```

Comment:
```suggestion
theorem getElem_umod_of_pos {n d : BitVec w} (hi : i < w) (hd : 0#w < d) :
```

Nit: `0 < _` is generally referred to as `pos` in names, rather than `lt` (at least, I assume the `of_lt` in the name refers to `hd`, since the general `getElem_umod` also has `hi`)